builder/vmware/common/vnc_config.go

```diff
@@ -0,0 +1,32 @@
+//go:generate struct-markdown
+package common
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/packer/common/bootcommand"
+	"github.com/hashicorp/packer/template/interpolate"
+)
+
+// ~> **Note:** For ESXi builds the `disable_vnc` option is ignored. For remote builds, VNC is no longer used to send
+// the keystrokes, USB scancodes is used instead.
+type VNCConfigWrapper struct {
+	bootcommand.VNCConfig `mapstructure:",squash"`
+}
+
+func (c *VNCConfigWrapper) Prepare(ctx *interpolate.Context, driverConfig *DriverConfig) (warnings []string, errs []error) {
+	if driverConfig.RemoteType != "" {
+		if !c.DisableVNC {
+			warnings = append(warnings, "[WARN] The vmware-esxi do not use VNC to connect to the host anymore. By default, the VNC is disabled.")
```

Comment:
```suggestion
			warnings = append(warnings, "[WARN] The remote VMWare builds do not use VNC to connect to the host anymore. The disable_vnc option will be ignored and automatically set to true.")
```

Reply:
side note -- I like how you've created this wrapper to test against options that are outside of the vnc config. Really clean. We should use this pattern elsewhere.